project.clj

```diff
@@ -58,12 +59,14 @@
                  [com.taoensso/nippy "2.13.0"]                        ; Fast serialization (i.e., GZIP) library for Clojure
                  [compojure "1.5.2"]                                  ; HTTP Routing library built on Ring
                  [crypto-random "1.2.0"]                              ; library for generating cryptographically secure random bytes and strings
+                 [distributions "0.1.3-SNAPSHOT"                      ; Probability distribution
+                  :exclusions [org.clojure/clojure]]
```

Comment:
What @senior said about the `jdistlib` dep applies to this too. Relying on a `-SNAPSHOT` dependency is risky AF because it could change out from under us without notice. Preferable to stick to something like `0.1.2` instead of an unfinalized snapshot